share/spice/board_game_geek/search/board_game_geek_search.js

```diff
@@ -0,0 +1,136 @@
+            script = $('[src*="/js/spice/board_game_geek/"]')[0],
+            source = $(script).attr("src"),
+            query = source.match(/board_game_geek\/search\/([^\/]+)/)[1];
+
+        // because of the XML parsing, a response with 1 item gets
+        // turned into an object rather than array
+        if (!(items instanceof Array)) {
+            items = [items];
+        }
+
+        // in some cases BGG returns FAR TOO MANY results, including expansions
+        // this is especially a problem with games with fan expansions,
+        // e.g. Ticket To Ride has 100+ results
+        //
+        // to eliminate most of the expansions, we get the earliest result by year
+        // (using the numeric ID as a tiebreaker) and grab the first 25 entries only
+        items.sort(function (a, b) {
+            var aYear = Number(a.yearpublished && a.yearpublished.value) || 5000,
+                bYear = Number(b.yearpublished && b.yearpublished.value) || 5000;
+
+            if (aYear === bYear) {
+                return Number(a.id) < Number(b.id) ? -1 : 1;
+            } else {
+                return aYear < bYear ? -1 : 1;
+            }
+        });
+
+        items = items.slice(0, 25);
+
+        // Render the response
+        Spice.add({
+            id: "board_game_geek",
+            name: "Board Games",
+            data: items,
+            meta: {
+                itemType: "Board Games",
+                searchTerm: decodeURIComponent(query),
+                sourceName: "BoardGameGeek",
+                sourceUrl: "http://boardgamegeek.com/geeksearch.php?action=search&objecttype=boardgame&q=" + query,
+                sourceIcon: true,
+                rerender: ["img", "img_m", "abstract", "rating", "reviewCount", "players", "playTime", "age"]
+            },
+            normalize: function(item) {
+                return {
+                    bggId: item.id, // store id to use it in onItemShown
+                    heading: item.name.value,
+                    img: "",
+                    img_m: "",
+                    url: "http://boardgamegeek.com/boardgame/" + item.id,
+                    url_review: "http://boardgamegeek.com/boardgame/" + item.id + "/reviews"
+                };
+            },
+            onItemShown: function (item) {
+                if (item.loadedDetails) {
+                    return;
+                }
+
+                $.getJSON("/js/spice/board_game_geek/get_details/" + item.bggId, function (response) {
+                    var responseItem = response.items.item,
+                        rating = DDG.getProperty(responseItem, "statistics.ratings.average.value");
+
+                    // the BGG rating is out of 10 so divide to get a five star rating
+                    if (rating) {
+                        rating /= 2;
+                    }
+
+                    item.set({
+                        img_m: DDG.getProperty(responseItem, "image.text"),
```

Comment:
@andrey-p looks like these URL's all start with `//` now. If we strip that, the images should work again 👍